webtau-groovy/src/main/groovy/com/twosigma/webtau/cfg/WebTauCliArgsConfig.groovy

```diff
@@ -62,16 +67,26 @@ class WebTauCliArgsConfig {
         Options options = createOptions()
         commandLine = createCommandLine(args, options)
 
-        if (commandLine.hasOption("help") || commandLine.argList.isEmpty()) {
-            HelpFormatter helpFormatter = new HelpFormatter()
-
-            def header = "version: " + WebTauMeta.version
-            helpFormatter.printHelp("webtau [options] [testFile1] [testFile2]", header, options, "")
-            exitHandler.exit(1)
-            return
+        if (commandLine.hasOption(EXAMPLE_OPTION)) {
+            scaffoldExamples()
+        } else if (commandLine.hasOption(HELP_OPTION) || commandLine.argList.isEmpty()) {
+            printHelp(options)
+        } else {
+            testFiles = new ArrayList<>(commandLine.argList)
         }
+    }
+
+    private void scaffoldExamples() {
+        ExamplesScaffolder.scaffold(Paths.get(""))
```

Comment:
Do we need the argument here?  We only ever seem to pass `Paths.get("")`.  If we do need it but this is a sensible default then I'd suggest adding an argument-less overload of `scaffold`.

Reply:
I was entertaining the possibility of handling `--workingDir` value in future